bin/main.ml

```diff
@@ -97,19 +97,50 @@ let version =
 
 let info = Cmd.info "get-activity" ~version
 
+module Client = struct
+  let null_auth ?ip:_ ~host:_ _ = Ok None
```

Comment:
We really should make a cohttp-eio-tls package with a proper authenticator.

Reply:
I can take a look at doing it this week.

Reply:
Thanks, but I'm only going to work on this PR during Hacking Days, so no pressure!

Reply:
I've updated the example to use a proper authenticator now: https://github.com/mirage/ocaml-cohttp/pull/1091

It's just:
```ocaml
let authenticator =
  match Ca_certs.authenticator () with
  | Ok x -> x
  | Error (`Msg m) -> Fmt.failwith "Failed to create system store X509 authenticator: %s" m
```